Gemfile

```diff
@@ -1,12 +1,14 @@
 source 'https://rubygems.org'
 
-gemspec
+gemspec :name => 'smart_proxy_dynflow_core'
 
 group :development do
   gem 'smart_proxy', :git => "https://github.com/theforeman/smart-proxy", :branch => "develop"
   gem 'pry'
 end
 
+gem 'smart_proxy_dynflow', :path => '.'
```

Comment:
ok, currently we pack this Gemfile in the gem which is causing problems, maybe we should wrap this in some $var condition and skip it in core gemspec